src/plugins/yamlcpp/README.md

```diff
@@ -186,34 +184,6 @@ kdb rm -r user/tests/yamlcpp
 sudo kdb umount user/tests/yamlcpp
 ```
 
-### Sparse Arrays
```

Comment:
Why are you removing this?

Reply:
This looks like rebase mistake from my side. I changed it accordingly